yt_dlp/extractor/itprotv.py

```diff
@@ -0,0 +1,174 @@
+            'description': 'Welcome to ITProTV! The guided tour you are about to participate in has been assembled to introduce you to ITProTV and show you many of the features available as part of ITProTV.',
+            'duration': 269,
+            'series': 'ITProTV 101',
+            'series_id': 'guided-tour',
+            'availability': 'needs_auth',
+            'chapter': 'ITProTV 101',
+            'chapter_number': 1,
+            'chapter_id': '5dbb3de426b46c0010b5d1b6'
+        },
+        'params': {
+            'skip_download': True,
+        },
+    },
+        {
+        'url': 'https://app.itpro.tv/course/beyond-tech/job-interview-tips',
+        'md5': '101a299b98c47ccf4c67f9f0951defa8',
+        'info_dict': {
+            'id': 'job-interview-tips',
+            'ext': 'mp4',
+            'title': 'Job Interview Tips',
+            'thumbnail': 'https://s3.amazonaws.com:443/production-itprotv-thumbnails/2f370bf5-294d-4bbe-ab80-c0b5781630ea.png',
+            'description': "Did you know that preparation is one of the most important aspects of a successful job interview? Just as the interviewer will do their research on you, it's equally as important for you to do your research on the company. In this episode Jo will share with you some key preparation tips and best practices that will hopefully help you nail your next job interview!",
+            'duration': 267,
+            'series': 'Beyond Tech',
+            'series_id': 'beyond-tech',
+            'availability': 'needs_auth',
+            'chapter': 'Job Development',
+            'chapter_number': 2,
+            'chapter_id': '5f7c78d424330c000edf04d9'
+        },
+
+    }
+    ]
+
+    def _get_course_api_json(self, webpage, course_name):
+        jwt = self._fetch_jwt(webpage)
+
+        if jwt:
+            headers = {'Authorization': 'Bearer ' + jwt}
+            course_api = self._download_json(
+                f'https://api.itpro.tv/api/urza/v3/consumer-web/course?url={course_name}&brand=00002560-0000-3fa9-0000-1d61000035f3',
+                course_name, headers=headers, note='Fetching data from course API')
+
+        return course_api
+
+    def _get_episode_api_json(self, webpage, episode_id):
+        jwt = self._fetch_jwt(webpage)
+
+        if jwt:
+            headers = {'Authorization': 'Bearer ' + jwt}
+            episode_api = self._download_json(
+                f'https://api.itpro.tv/api/urza/v3/consumer-web/brand/00002560-0000-3fa9-0000-1d61000035f3/episode?url={episode_id}',
+                episode_id, headers=headers, note='Fetching data from episode API')
+
+        return episode_api
+
+    def _fetch_jwt(self, webpage):
+        return self._search_regex(r'{"passedToken":"([A-Za-z0-9-_]*\.[A-Za-z0-9-_]*\.[A-Za-z0-9-_]+)",', webpage, 'jwt')
+
+    def _check_if_logged_in(self, webpage):
+        if '{ member: null' in webpage:
+            raise self.raise_login_required(method='cookies')
```

Comment:
```suggestion
            raise self.raise_login_required()
```
due to some recent changes, method is now automatically determined